### ISSUE TITLE
latest version doesn't install correctly.

### ISSUE THREAD

**Issue by KRRT7** [darrenburns/rich-pixels#17]
```
❯ pip show rich-pixels
Name: rich-pixels
Version: 3.0.0
Summary: A Rich-compatible library for writing pixel images and ASCII art to the terminal.
Home-page:
Author:
Author-email: Darren Burns <darrenb900@gmail.com>
License:
Location: venv\Lib\site-packages
Requires: pillow, rich
Required-by:
```

```
❯ py -c "import pygame"
Traceback (most recent call last):
  File "<string>", line 1, in <module>
ModuleNotFoundError: No module named 'pygame'
```
installed via pip install rich-pixels, 2.2.0 works perfectly

**Comment by darrenburns** [comment]
The sample snippet you gave is trying to import pygame?

**Comment by KRRT7** [comment]
oops, i don't know what happened there, most likely lack of sleep, anyways
```
░▒▓ 🎭  …\Desktop\OSS contributions\rich-pixels-MRE   10:26 
❯ pip install rich-pixels
Collecting rich-pixels
  Using cached rich_pixels-3.0.0-py3-none-any.whl.metadata (2.2 kB)
Collecting pillow>=10.0.0 (from rich-pixels)
  Using cached pillow-10.2.0-cp311-cp311-win_amd64.whl.metadata (9.9 kB)
Collecting rich>=12.0.0 (from rich-pixels)
  Using cached rich-13.7.0-py3-none-any.whl.metadata (18 kB)
Collecting markdown-it-py>=2.2.0 (from rich>=12.0.0->rich-pixels)
  Using cached markdown_it_py-3.0.0-py3-none-any.whl.metadata (6.9 kB)
Collecting pygments<3.0.0,>=2.13.0 (from rich>=12.0.0->rich-pixels)
  Using cached pygments-2.17.2-py3-none-any.whl.metadata (2.6 kB)
Collecting mdurl~=0.1 (from markdown-it-py>=2.2.0->rich>=12.0.0->rich-pixels)
  Using cached mdurl-0.1.2-py3-none-any.whl (10.0 kB)
Using cached rich_pixels-3.0.0-py3-none-any.whl (2.6 kB)
Using cached pillow-10.2.0-cp311-cp311-win_amd64.whl (2.6 MB)
Using cached rich-13.7.0-py3-none-any.whl (240 kB)
Using cached markdown_it_py-3.0.0-py3-none-any.whl (87 kB)
Using cached pygments-2.17.2-py3-none-any.whl (1.2 MB)
Installing collected packages: pygments, pillow, mdurl, markdown-it-py, rich, rich-pixels
Successfully installed markdown-it-py-3.0.0 mdurl-0.1.2 pillow-10.2.0 pygments-2.17.2 rich-13.7.0 rich-pixels-3.0.0


░▒▓ 🎭  …\Desktop\OSS contributions\rich-pixels-MRE   10:27 
❯ py -c "import rich_pixels"
Traceback (most recent call last):
  File "<string>", line 1, in <module>
ModuleNotFoundError: No module named 'rich_pixels'
```

**Comment by darrenburns** [comment]
Interesting - I did switch from Poetry to Rye, so maybe I made a mistake somewhere. Thanks for the report. I'll try to have a look later tonight.

**Comment by wilderrodrigues** [comment]
I just got 3.0.0 with Poetry and it's not working. It installs, but the module cannot be found. Reverted to 2.2.0 and all is fine.

**Comment by kitakitaaura** [comment]
Yup, having the same issue

**Comment by darrenburns** [comment]
Should be fixed in 3.0.1 - sorry for the delay.
